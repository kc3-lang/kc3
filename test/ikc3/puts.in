quote puts("Test")
puts("Test")
quote puts(%{hello: "World !"})
puts(%{hello: "World !"})
