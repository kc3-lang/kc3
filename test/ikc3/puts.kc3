quote puts("Test")
puts("Test")
quote puts(%{hello: "World !"})
puts(%{hello: "World !"})
quote puts("Test #{:a}")
puts("Test #{:a}")
