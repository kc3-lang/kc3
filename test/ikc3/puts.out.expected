puts("Test")
Test
5
puts(%{hello: "World !"})
%{hello: "World !"}
20
